core/storage/storage_models_test.py

```diff
@@ -101,10 +103,26 @@ def test_all_child_models_have_get_deletion_policy(self):
                 self.fail(msg='get_deletion_policy is not defined for %s' % (
                     clazz.__name__))
 
-    def test_all_base_models_do_not_have_get_deletion_policy(self):
-        model_subclasses = self._get_model_classes()
-
-        for clazz in model_subclasses:
+    def test_base_models_do_not_have_get_deletion_policy(self):
+        for clazz in self._get_model_classes():
             if clazz.__name__ in self.BASE_CLASSES:
                 with self.assertRaises(NotImplementedError):
                     clazz.get_deletion_policy()
+
+    @unittest.skip(
+        'has_reference_to_user_id is not yet implemented on all models')
+    def test_base_or_versioned_child_classes_have_has_reference_to_user_id(
+            self):
+        for clazz in self._get_base_or_versioned_model_child_classes():
+            try:
+                self.assertIsNotNone(clazz.has_reference_to_user_id('any_id'))
+            except NotImplementedError:
+                self.fail(
+                    msg='has_reference_to_user_id is not defined for %s' % (
+                        clazz.__name__))
+
+    def test_base_models_do_not_have_has_reference_to_user_id(self):
```

Comment:
..._have_has_... sounds weird, probably better naming here and above could be:
```suggestion
    def test_base_models_do_not_have_method_has_reference_to_user_id(self):
```